### DISCUSSION TITLE
Welcome to CusADi Discussions!

### DISCUSSION THREAD

**Discussion by se-hwan** [se-hwan/cusadi#3] [Announcements]
## 👋 Welcome!

Please feel free to post questions about the CasADi, the paper, or details of implementation here.
If there's a clear bug with the codegen or CUDA, feel free to make a separate`Issue` as well.

**Comment by kassasin** [comment]
Excellent work,Is there some examples as the video showed for humanoid robot simulation and learning ？

**Comment by lvjonok** [comment]
Thank you, Se-hwan, for bringing CusADi into the community! 🙌

Inspired by the great work done with CusADi, we have developed a new library called **JaxADi**. It inherits the core concepts of CusADi but is built using JAX, providing more user-friendly features and multi-platform support compared to CUDA and Torch. As a side-effect, we have worked on optimizing code generation and reduced the number of code lines generated by up to 2X. We hope JaxADi can serve as a complementary tool for those looking for similar functionality with the advantages of JAX.

We invite everyone to take a look and share your feedback!

🔗 [JaxADi on GitHub](https://github.com/based-robotics/jaxadi)

Looking forward to your thoughts and contributions!
